platform-tests/src/platform/acceptance/uaa_styling_test.go

```diff
@@ -0,0 +1,71 @@
+package acceptance_test
+
+import (
+	"encoding/json"
+	"github.com/PuerkitoBio/goquery"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"io/ioutil"
+	"net/url"
+)
+
+var _ = Describe("UAA Styling", func() {
+	var uaaURL *url.URL
+	var uaaLoginURL url.URL
+	var uaaStylesheetURL url.URL
+	var customStylesheetURL url.URL
+
+	BeforeEach(func() {
+		infoCommand := cf.Cf("curl", "/v2/info")
+		Expect(infoCommand.Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
+
+		var infoResp struct {
+			TokenEndpoint string `json:"token_endpoint"`
```

Comment:
It might be a bit of a moot point because they both serve the same content, but I think this should use `authorization_endpoint` (`login.system_domain`) rather than `token_endpoint` (`uaa.system_domain`) to be consistent with some other places:

- The URL returned by `cf login --sso`
- The URL sent to users in invites: https://github.com/alphagov/paas-cf/blob/c17ba3bcd3270158a581524456dba22e4b4f92ee/scripts/create-user.sh#L152